Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -644,18 +644,20 @@ theorem coe_inclusion (s : S) : (inclusion h s : A) = s :=
 
 namespace inclusion
 
-scoped instance (X) [SMul X R] [SMul X A] [IsScalarTower X R A] : letI := (inclusion h).toModule
-    IsScalarTower X S T :=
+scoped instance instIsScalarTowerSubtypeMemBot (X) [SMul X R] [SMul X A] [IsScalarTower X R A] :
```

Comment:
It should be snake_case because it is a Prop.

Reply:
Well, I try not to deviate too much from the auto-generated name. @eric-wieser may have something to say about the convention to be followed here.

Reply:
The current state is like this; I'm yet to find a single instance with a snake-cased name.
![image](https://github.com/user-attachments/assets/ba5bea4e-4f11-4394-be89-879b94313ef6)
![image](https://github.com/user-attachments/assets/a68f75cd-21d1-4215-9426-b24b4856e4a1)

Reply:
I am pretty certain that most `IsScalarTower` that is given an explicit name is in snake case (e.g. `RingCon.isScalarTower_right` or `IsScalarTower.of_algHom` or `Basis.isScalarTower_of_nonempty` or `Ideal.Quotient.isScalarTower_of_liesOver` or `Ideal.Quotient.tower_quotient_map_quotient` or `IsLocalization.localization_isScalarTower_of_submonoid_le`) and even if they are, they should be according to the naming convention.

Reply:
But I am happy to have a third opinion. Either you could wait for Eric to comment here or raise a topic on Zulip. The `awaiting-author` is for the build error and is unrelated to this.

Reply:
Thanks for the examples; I've renamed the instances.